apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -91,26 +125,7 @@ export function createDefaultSpriteMetadata(listData) {
 // Regenerates the metadata for the default list of sprites in SpriteLab
 export function regenerateDefaultSpriteMetadata(listData) {
   return createDefaultSpriteMetadata(listData).then(defaultMetadata => {
-    return fetch(`/api/v1/animation-library/default-spritelab-metadata`, {
-      method: 'POST',
-      headers: {
-        'content-type': 'application/json'
-      },
-      body: JSON.stringify(defaultMetadata)
-    })
-      .then(response => {
-        if (!response.ok) {
-          throw new Error(
-            `Default Sprite Metadata Upload Error(${response.status}: ${
-              response.statusText
-            })`
-          );
-        }
-        return Promise.resolve();
-      })
-      .catch(err => {
-        return Promise.reject(err);
-      });
+    return uploadDefaultListMetadata(defaultMetadata, 'levelbuilder');
```

Comment:
what should we do if `createDefaultSpriteMetadata` fails?

Reply:
Added a catch branch.